buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -136,7 +152,10 @@ public class PluginBuildPlugin extends BuildPlugin {
     private static void createIntegTestTask(Project project) {
         RestIntegTestTask integTest = project.tasks.create('integTest', RestIntegTestTask.class)
         integTest.mustRunAfter(project.precommit, project.test)
-        project.integTestCluster.distribution = System.getProperty('tests.distribution', 'integ-test-zip')
+        if (project.plugins.hasPlugin(TestClustersPlugin.class) == false) {
```

Comment:
Again, I'd put this in a lazy plugin application block which goes back and removes this system property rather than rely on the test clusters plugin to have already been applied.